www/map-view.js

```diff
@@ -66,16 +66,18 @@ class MapView extends HTMLElement {
   connectedCallback() {
     MapViewController._prepare(this);
     MapViewController._getDeviceId((deviceId) => {
-      this.innerHTML = `
-        <iframe
-          id="map-view-iframe"
-          src="${this._getViewerURL(deviceId)}"
-          width="100%"
-          height="100%"
-          style="border: none; box-shadow: none;"
-        />
-      `;
+      const iframe = document.createElement('iframe');
```

Comment:
Este cambio evita un dos reportes de semgrep. Non lle gustaba que o src do iframe se establecera "de calquera maneira", parece que invocando setAttribute é máis seguro (aínda que non me ten moita lóxica).